src/sas/qtgui/Perspectives/Fitting/FittingUtilities.py

```diff
@@ -155,11 +158,15 @@ def addParametersToModel(model, view, parameters, kernel_module, is2D):
         cbox = createFixedChoiceComboBox(param, row)
 
         # Append to the model and use the combobox, if required
-        model.appendRow(row)
-        if cbox is not None:
-            view.setIndexWidget(item2.index(), cbox)
+        if None not in (model, view):
```

Comment:
This snippet is identical to the others in this commit - would be great if refactored